lib/hackerone/client.rb

```diff
@@ -52,6 +55,44 @@ def program
         @program || HackerOne::Client.program
       end
 
+      def programs
+        response = self.class.hackerone_api_connection.get do |req|
+          req.url "me/programs"
+        end
+
+        data = JSON.parse(response.body, :symbolize_names => true)[:data]
+        if data.nil?
+          raise RuntimeError, "Expected data attribute in response: #{response.body}"
+        end
+
+        data.map do |program|
+          Program.new(program)
+        end
+      end
+
+      def program_as_object
```

Comment:
What do you think about moving this into something like `Program.find(program)`? It will be a little strange that the "model" class would end up making an API call, but I think that's OK.